napari/_qt/widgets/qt_progress_bar.py

```diff
@@ -13,9 +13,12 @@
 class ProgressBar(QWidget):
```

Comment:
```suggestion
class QtLabeledProgressBar(QWidget):
```

As this object inherits from a `QWidget`, maybe it should be named `Qt...`